### PR TITLE
Point people to sugardough instead

### PR DESCRIPTION
**Pull request by tofumatt**
Sweeter! :cookie:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
 
-Mozilla's Playdoh is a web application template based on [Django][django].
-
-Patches are welcome! Feel free to fork and contribute to this project on
-[github][gh-playdoh].
-
-Full [documentation][docs] is available as well.
+_playdoh_ is Mozilla's old Django app template, but it has been replaced by [sugardough][], which you should use instead.
 
+The [old documentation][docs] for playdog is available if you need it.
```
Comment (pmclanahan) [inline README.md]: `s/playdog/playdough/`

Comment (tofumatt) [inline README.md]: s/playdough/playdoh

zing!

**Comment by pmclanahan** [comment]
r+ from me.

**Comment by glogiotatidis** [comment]
\o/!

**Comment by kumar303** [comment]
:+1: :zap:

**Pull request merged**